rules/properImportRequireRule.ts

```diff
@@ -0,0 +1,136 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import * as Lint from 'tslint';
+import * as ts from 'typescript';
+
+// True if es6 module
+const module_cache: { [k: string]: boolean } = {};
+
+function findModulePath(module_name: string): string {
+  let current_dir = process.cwd();
+  if (module_name.startsWith('./')) return ''; // Currently doesn't support local modules.
```

Comment:
Why not?

Reply:
`tslint` would checking typescript files rather than the compiled versions. Doesn't work with the current solution.